blenderproc/python/loader/Front3DLoader.py

```diff
@@ -416,4 +419,32 @@ def _move_and_duplicate_furniture(data: dict, all_loaded_furniture: list) -> Lis
                             rotation_mat = mathutils.Quaternion(child["rot"]).to_euler().to_matrix().to_4x4()
                             # transform it into the blender coordinate system and then to an euler
                             new_obj.set_rotation_euler((blender_rot_mat @ rotation_mat).to_euler())
+                            
+                            if new_obj.get_cp("uid") == prev_uid:
+                                equal_objs[-1].append(new_obj)
+                            else:
+                                equal_objs.append([new_obj])
+                                prev_uid = new_obj.get_cp("uid")
+        for i in equal_objs:
+            if len(i) == 1:
+                created_objects.append(i[0])
+                bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY', center='MEDIAN')
```

Comment:
Why are you changing the origin here?

Reply:
mainly to get the lights in the right location within the object